modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -185,6 +193,11 @@ export default class SVMClassifier {
             })
           }
 
+          oosPoints.push({
```

Comment:
Another idea which would need further testing would be to use words in the given language but are clearly OOV. Using randomly picked but real world words, computing the distance, with having a minimum distance to be set as OOV features plus the none utterances. This would help svm to scope even more around vocab